docs/compilers/CSharp/Compiler Breaking Changes - post VS2017.md

```diff
@@ -14,3 +14,8 @@ For instance, `var t = (a, b.c, this.d);` will produce a tuple with element name
 Consider the case where the type of `a` is `System.Func<bool>` and you write `var local = t.a();`. This will now find the first element of the tuple and invoke it, whereas previously it could only mean "invoke an extension method named 'a'".
 
 - https://github.com/dotnet/roslyn/issues/16870 In C# 7.0 and before C# 7.1, the compiler accepted self-assignments in deconstruction-assignment. The compiler now produces a warning for that. For instance, in `(x, y) = (x, 2);`.
+
+- The compiler is now more precise in detecting erroneous pattern-matching operations because the expression could not possibly match the pattern. The following situations now cause an error:
```

Comment:
There is only one issue, and it doesn't have more information, but I'll add a link.